packages/hoprd/src/index.ts

```diff
@@ -194,8 +194,6 @@ async function main() {
       })
     }
 
-    node.on('hopr:message', logMessageToNode)
```

Comment:
Why is this included?

Reply:
it's a duplicate afaik

Reply:
https://github.com/hoprnet/hoprnet/blob/master/packages/hoprd/src/index.ts#L171